src/sx-tailwind-website/src/pages/_document.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 
-import type { Node, Element } from 'react';
-import Document, { Head, Main, NextScript, type DocumentContext } from 'next/document';
+import type { Node } from 'react';
+// $FlowExpectedError[missing-export]: Flow-typed types needs updating
```

Comment:
Just a detail: probably not `$FlowExpectedError` but rather `$FlowFixMe` (it's not an expected error, it's a problem that needs to be fixed). 😎

Reply:
I most certainly expected this error, as it didn't have a flow type. It might be a problem that needs to be fixed, but it needs to be fixed outside of the `@aderia` scope. I know we can submit a PR for it, but still, from the top of my head, `FlowExpectedError` seems more appropriate here.